core/templates/pages/contributor-dashboard-page/services/contribution-and-review-backend-api.service.spec.ts

```diff
@@ -0,0 +1,306 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for contribution and review backend api service.
+ */
+
+import { HttpClientTestingModule, HttpTestingController }
+  from '@angular/common/http/testing';
+import { TestBed, fakeAsync, flushMicrotasks } from '@angular/core/testing';
+import { ContributionAndReviewBackendApiService }
+  from './contribution-and-review-backend-api.service';
+
+describe('Contribution and review backend API service', () => {
+  let carbas: ContributionAndReviewBackendApiService;
+  let http: HttpTestingController;
+
+  let suggestion1 = {
```

Comment:
In general, prefer `const` over `let` if the variable is not being reassigned. Here and below.

Reply:
done